docs/source/api-main-setup.rst

```diff
@@ -0,0 +1,167 @@
+==================================
+Lookit API Local Machine Setup
+==================================
+
+Prerequisites
+~~~~~~~~~~~~~
+Before you can get the API running on your machine, there are a few steps you need to take to ensure everything will work
```

Comment:
example of where I might not call this "the API." (honestly to me it seems like an odd name for the repo altogether but maybe that's just me)